db/migrate/20160105151541_devise_create_users.rb

```diff
@@ -6,7 +6,7 @@ def change
       t.string :username, null: false
       t.string :email,              null: false, default: ""
       t.string :encrypted_password, null: false, default: ""
-
+      t.string :role, null: false, default: "member"
```

Comment:
change this to an `admin` attribute which is a boolean. Then you can do things like `current_user.admin?`